pkg/reconciler/openshift/tektonaddon/tektonaddon.go

```diff
@@ -56,7 +59,17 @@ type Reconciler struct {
 var _ tektonaddonreconciler.Interface = (*Reconciler)(nil)
 var _ tektonaddonreconciler.Finalizer = (*Reconciler)(nil)
 
-var addonResourceName = "addon"
+var communityResourceURLs = []string{
```

Comment:
Should we move this somewhere else?

Reply:
yes, we could :+1: ,
but let us handle it in a future refactoring.